docs/5.0/kubernetes-access.md

```diff
@@ -5,61 +5,244 @@ description: How to set up and configure Teleport for Kubernetes access with SSO
 
 # Teleport Kubernetes Access
 
-Teleport has the ability to act as a compliance gateway for managing privileged
-access to Kubernetes clusters. This enables the following capabilities:
+Teleport manages privileged access to Kubernetes clusters.
 
-* Teleport acts as a single authentication endpoint for both SSH and multiple
-  Kubernetes clusters. Users can authenticate against a Teleport Access Plane using
-  Teleport's [`tsh login`](cli-docs.md#tsh-login) command and retrieve credentials
-  for both SSH and Kubernetes API.
-* Users RBAC roles are always synchronized between SSH and Kubernetes, making
-  it easier to implement policies like _developers must not access production
-  data_.
-* Complete `kubectl` auditing and session recordings for `kubectl exec`
-* Multi Kubernetes Support. Login to Teleport once and quickly switch between multiple K8s clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login)
+* Users can login once using [`tsh login`](cli-docs.md#tsh-login) and
+  switch between multiple clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login).
+* Admins can use roles to implement policies like `developers must not access production`.
+* Teleport captures `kubectl` events and session recordings for `kubectl exec`.
 
-## Start Here
+## Deploy Teleport In Kubernetes
 
-Before we dive into setup, we've a few options to help guide you. We've created some example
-Teleport Kubernetes Access configurations for different scenarios, all the way from solo developers
-accessing minikube through to large enterprises accessing hundreds of Kubernetes clusters.
+![teleport-kubernetes-inside](img/teleport-k8s-pod.svg)
+
+=== "Community"
 
-Teleport Kubernetes service requires Auth & Proxy to be setup. This can be inside or
-outside of k8s.
+    Setup and Install Teleport Helm chart repository.
+    ```bash
+    helm repo add teleport https://charts.releases.teleport.dev
+    helm install teleport teleport/teleport
+    ```
+
+    Download Demo Helm Value and Update Teleport Pod.
+    ```bash
+    curl https://github.com/gravitational/teleport/blob/master/examples/chart/teleport/quickstart/values-community.yaml --output values.yaml
+    helm upgrade -f values.yaml teleport teleport/teleport
+    ```
 
-Example Kubernetes Cluster Configurations:
+    Check that the Teleport Service is running.
```

Comment:
this does not check that service is running, this checks if the service (load balancer) has been created. I think we can use `kubectl describe deployments/teleport`